client/components/ReviewRating/ReviewList/ReviewBody.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import React, { useState, useEffect } from 'react';
+import ReactModal from 'react-modal';
+
+const ReviewBody = ({ review }) => {
+  const [isMoreThanMaxChar, setIsMoreThanMaxChar] = useState(false);
+  const [isImageOpen, setIsImageOpen] = useState(false);
+  const [imageUrl, setImageUrl] = useState('');
+  const imgStyles = {
+    content: {
+      top: '50%',
+      left: '50%',
+      right: 'auto',
+      bottom: 'auto',
+      marginRight: '-50%',
+      transform: 'translate(-50%, -50%)',
+      width: '400px',
+      height: '405px',
+    },
+  };
+  const onClickImage = (e) => {
+    setIsImageOpen(!isImageOpen);
+    setImageUrl(e.target.src);
+  };
+  if (process.env.NODE_ENV !== 'test') ReactModal.setAppElement('#app');
+  useEffect(() => {
+    if (review && review.body.length > 250) {
+      setIsMoreThanMaxChar(true);
+    }
+  }, []);
+  return (
+    <div className="reviewBody" data-testid="reviewBody">
+      {isMoreThanMaxChar ? (
+        <div>
+          {review.body.slice(0, 250)}
+          <button type="button" onClick={() => setIsMoreThanMaxChar(false)}>show more</button>
+        </div>
+      )
+        : <div>{review && review.body}</div>}
+      {
+        review && review.photos.map((photo) => (
+          <>
```

Comment:
When you change your App.jsx to another id like `  {product && <ReviewRating id={product[2].id} />}`, you'll get the warning key is not unique. You might want to remove those <> </> fragments, since you only need to return the img div and that fixes the issue.